interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -226,6 +242,21 @@ namespace cling {
     DiagnosticConsumer& DClient = getCI()->getDiagnosticClient();
     DClient.BeginSourceFile(getCI()->getLangOpts(), &PP);
 
+    if (m_IncrParser->getCI()->getLangOpts().Modules) {
```

Comment:
Why not using `usingCxxModules` and moving the next if statement here?

Reply:
DOne.